src/main/java/racingcar/view/InputView.java

```diff
@@ -0,0 +1,100 @@
+package racingcar.view;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import camp.nextstep.edu.missionutils.Console;
+
+public class InputView {
+    private static final String REQUEST_INPUT_CAR_NAMES = "경주할 자동차 이름을 입력하세요.(이름은 쉼표(,) 기준으로 구분)";
+    private static final String REQUEST_INPUT_TRY_COUNT = "시도할 회수는 몇회인가요?";
+    private static final String COMMA_DELIMITER = ",";
+
+    private static final String ERROR_WHITE_SPACE_NAME = "[ERROR] 공백인 이름은 입력할 수 없습니다. 다시 입력해주세요.";
+    private static final String ERROR_NAME_LENGTH = "[ERROR] 길이가 5 초과인 이름은 입력할 수 없습니다. 다시 입력해주세요.";
+    private static final String ERROR_DUPLICATE_NAME = "[ERROR] 동일한 이름은 입력할 수 없습니다. 다시 입력해주세요.";
+    private static final String ERROR_TRY_COUNT_LETTER = "[ERROR] 시도 횟수는 숫자만 입력할 수 있습니다. 다시 입력해주세요.";
+    private static final String ERROR_TRY_COUNT_RANGE = "[ERROR] 시도 횟수는 1이상의 숫자만 입력할 수 있습니다. 다시 입력해주세요.";
+
+    public List<String> inputCarNames() {
+        System.out.println(REQUEST_INPUT_CAR_NAMES);
+        List<String> carNames = splitAndTrimInputCarNames();
+        while (!isValidInputCarNames(carNames)) {
```

Comment:
조건문에서 검증을 위해 `!` 를 사용하는건 안좋습니다.
실수로 ! 를 놓치고 읽을 수도 있고 가독성 측면에도 좋지않습니다.
해당 메서드 내부에서 `!` 연산을 통해 false 를 반환하는 메서드로 로직을 변경해보는건 어떨까요?

```suggestion
        while (isNotValidInputCarNames(carNames)) {
```

Reply:
`!`을 지양하도록 해야겠군요! 
좋은 말씀 감사합니다!!